pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1303,6 +1303,20 @@ func parseDeviceIoListConfig(getconfigCtx *getconfigContext,
 					"key: %s, value: %s", key, value)
 			}
 		}
+
+		if ioDevicePtr.Vflist != nil && ioDevicePtr.Vflist.VfCount > 0 {
+			port.Vfs.Count = uint8(ioDevicePtr.Vflist.VfCount)
+			port.Vfs.Data = make([]sriov.EthVF, ioDevicePtr.Vflist.VfCount)
+			for i, vf := range ioDevicePtr.Vflist.Data {
+				port.Vfs.Data[i] = sriov.EthVF{
+					Index:   uint8(vf.Index),
+					Mac:     vf.Mac,
+					VlanID:  uint16(vf.VlanId),
```

Comment:
We should check if the configured VlanId is valid, like we do [here](https://github.com/lf-edge/eve/blob/9.4/pkg/pillar/cmd/zedagent/parseconfig.go#L659-L663).

Reply:
ah, good idea